plugins/org.obeonetwork.dsl.uml2.design/src/org/obeonetwork/dsl/uml2/design/api/services/ClassDiagramServices.java

```diff
@@ -81,6 +90,81 @@ public String computeAssociationEndLabel(Association association) {
 	}
 
 	/**
+	 * Compute label forn_ary association edge.
+	 *
+	 * @param association
+	 * @param view
+	 *            edge
+	 * @return label
+	 */
+	public String computeAssociationNAryBeginLabel(Association association, DDiagramElement view) {
```

Comment:
This method should just call a LabelServices#computeNAryAssociationBeginLabel method